Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/DirectionalLightComponentConfig.cpp

```diff
@@ -55,21 +56,15 @@ namespace AZ
             case PhotometricUnit::Lux:
                 return 0.0f;
             case PhotometricUnit::Ev100Illuminance:
-                return -10.0f;
+                return AZStd::numeric_limits<float>::lowest();
```

Comment:
Interesting, this may be the first time I've ever seen lowest() and not -FLT_MAX or something like that.